engines/producers/app/views/layouts/producers/application.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Producers</title>
+  <%= stylesheet_link_tag    "producers/application", media: "all" %>
+  <%= javascript_include_tag "producers/application" %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+
+<%= yield %>
+
+</body>
+</html>
```

Comment:
I was thinking about how to organize UI interface. Maybe we could create an interface engine and put all visuals there (rails view/controllers, javascripts, css,...)

And leave the rest of engines as services used in Interface engine controllers... I not sure splitting UI in engines is a good aproach.

Let's discuss this offline  in some moment.

/cc @enricostano @sauloperez